docs/react/api-angular/schematics/karma-project.md

```diff
@@ -6,7 +6,20 @@ Add karma testing to a project
 
 ```bash
 nx generate karma-project ...
+```
+
+By default, Nx will search for `karma-project` in the default collection provisioned in `workspace.json`.
+
+You can specify the collection explicitly as follows:
+
+```bash
+nx g @nrwl/angular:karma-project ...
+```
 
+Show what will be generated without writing to disk:
+
+```bash
+nx g null ... --dry-run
```

Comment:
There is still null here. This should be ng g downgrade-module or ng g @nrwl/angular:downgrade-module --dry-run